Update progress-plugin.md (#3554)
webpack/webpack.js.org: src/content/plugins/progress-plugin.md

});
```

> ## Percent calculation

Reviewer: ```suggestion
## Percentage calculation
```

> By default, progress percentage calculates based on built modules count and total modules count: `built / total`

Reviewer: ```suggestion
By default, progress percentage is calculated based on built modules count and total modules count: `built / total`
```

The total modules count is unknown in advance and changes during the build. This may cause inaccurate progress percentage.

> To solve this problem `ProgressPlugin` caches the last know total modules count and reuses this value on the next build. The first build will warm the cache but the following builds will use and update this value.

Reviewer: ```suggestion
To solve this problem `ProgressPlugin` caches the last known total modules count and reuses this value on the next build. The first build will warm the cache but the following builds will use and update this value.
```
